### ISSUE TITLE
Deprecated keyring warning in `apt-get`

### ISSUE THREAD

**Issue by silverwind** [nikolaik/docker-python-nodejs#114]
Running `apt-get update` in the new debian 12 image gives these two warnings:

```
W: https://dl.yarnpkg.com/debian/dists/stable/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.

W: https://deb.nodesource.com/node_20.x/dists/bookworm/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
```

**Comment by silverwind** [comment]
Ah I see https://github.com/nikolaik/docker-python-nodejs/pull/109 which might fix it.

**Comment by nikolaik** [comment]
Thanks for reporting @silverwind <3 I have an experimental build in https://github.com/nikolaik/docker-python-nodejs/pull/109 you append `-canary` to the tag you are using, f.ex. `python3.11-nodejs20-canary`. Does it break in anyway?

**Comment by silverwind** [comment]
No, does not break, just a warning that can be ignored. Might test later.

**Comment by silverwind** [comment]
BTW nodesource has recently updated their install instructions. IIRC, with those apt will not warn:

https://github.com/nodesource/distributions#nodejs

**Comment by nikolaik** [comment]
Thanks for checking 😊 Yup, that's nice of them! There is a more compelling reason to move away from nodesource debs than a mere warning and that is to avoid installing an extra python version, since the models package has a dependency on the distro packaged `python-minimal`

**Comment by silverwind** [comment]
Yes, I don't see python needed at all for node development. Wonder why they keep the dependency around.

**Comment by nikolaik** [comment]
Fixed in #139 and #109
